src/GitHubApiClient.js

```diff
@@ -152,15 +152,31 @@ class GitHubApiClient {
     return team ? team.members.nodes.map((member) => member.login): []
   }
 
-  async getAllPullRequests({author, repo, user}) {
+  async getAllPullRequests(conditions) {
+
+    const author = conditions['author']
+    const repo = conditions['repo']
+    const user = conditions['user']
+    const org = conditions['org']
+    const reviewRequested = conditions['review-requested']
+
     author.values = _(await Promise.all(
       author.values.map((author) => {
         return this.isTeam(author) ? this.getTeamMembers(author) : author
       })
     )).flatten()
       .uniq()
       .value()
-    const prs = await this.getPullRequestsForAuthors(author, repo, user)
+
+    reviewRequested.values = _(await Promise.all(
+      reviewRequested.values.map((reviewer)=> {
+        return this.isTeam(reviewer) ? this.getTeamMembers(reviewer) : reviewer
```

Comment:
For PRs with requests for a specified team [GitHub offers](https://github.blog/2017-08-24-easily-filter-review-requests-by-team/) a `team-review-requested:TEAMNAME` qualifier